ccpp/data/GFS_typedefs.meta

```diff
@@ -5739,12 +6027,42 @@
   units = flag
   dimensions = ()
   type = integer
+[bl_mynn_closure]
+  standard_name = control_for_closure_level_in_mellor_yamada_nakanishi_niino_pbl_scheme
+  long_name = flag to determine the closure level for the mynn
+  units = flag
```

Comment:
According to the naming rules, flag is reserved for logical, so the unit should be 1 (even though it may be less clear)